### PR TITLE
Ignore blank pws

### PR DESCRIPTION
**Pull request by ryman1**

### CODE REVIEW

File: cracked_hash_stats.py [hunk 1]
```diff
@@ -122,14 +124,11 @@ def helpmsg():
...
 showCombinedStats = False  # Combine stats of both modern and history passwords
 showUncracked = False  # Print usernames with uncracked passwords
-
-if len(sys.argv) < 4:  # If no options are specified, use default options
-    showModernStats = True
-    showHistoryStats = True
+ignoreBlankPWUsers = True  # Ignore users whose hash == blank password
 
 try:
     opts, args = getopt.getopt(sys.argv[1:], 'hm:pc:CMHu',
```
Comment (bandrel) [inline cracked_hash_stats.py]: if you are adding options then you need to add them to the getopt.getopt(sys.argv[1:], .... section.

if the option does not have an argument then dont add a colon or = sign.

File: cracked_hash_stats.py [hunk 2]
```diff
@@ -155,6 +154,8 @@ def helpmsg():
         showHistoryStats = True
     elif opt in ('-u', '--uncracked'):
         showUncracked = True
+    elif opt in ('b', '--blank'):
```
Comment (bandrel) [inline cracked_hash_stats.py]: should be elif opt in ('-b', '--blank'):
not elif opt in ('b', '--blank'):

**Comment by bandrel** [comment]
Added inline notes.  Fix the issues and I'll look at it later

**Comment by ryman1** [comment]
Yeah it notifies when you do inline notes. The changes have been made.

**Comment by bandrel** [comment]
:+1:

**Comment by ryman1** [comment]
:godmode:

**Pull request merged**